modules/virt-about-runstrategies-vms.adoc

```diff
@@ -0,0 +1,52 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virt-create-vms.adoc
+
+[id="virt-about-runstrategies-vms_{context}"]
+= About RunStrategies for virtual machines
+
+Virtual machines have a `RunStrategy` setting to determine whether or not a guest should be running a virtual machine instance.
+
+There are four defined RunStrategies:
+
+* Always: A VirtualMachineInstance is always present. A new VirtualMachineInstance spawns if the original crashes.
```

Comment:
's/VirtualMachineInstance/virtual machine instance/' in all instances. The former is the object name but we're speaking generally here.
Alternatively if you declare it in the opening para, we could use 'VMI'.